pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/DSMServer.java

```diff
@@ -112,6 +112,7 @@
 import org.broadinstitute.dsm.route.TriggerSurveyRoute;
 import org.broadinstitute.dsm.route.UserSettingRoute;
 import org.broadinstitute.dsm.route.ViewFilterRoute;
+import org.broadinstitute.dsm.route.dashboard.NewDashboardRoute;
```

Comment:
I am not sure I like `New` can we name it `ConfigurableDashboardRoute` or something else. Or just `DashboardRoute` and rename the "old" one to `BeGoneDashboard…` 😂

Reply:
let's keep it as it is for now, once we fully remove the current dashboard then we can easily rename it